doc/nrf/ug_thingy91.rst

```diff
@@ -0,0 +1,262 @@
+######################
+
+The |NCS| provides support for developing on the Thingy:91 using the following inbuilt components:
+
+* nRF9160 System in Package (SiP)
+* nRF52840 System on Chip (SoC)
+
+Thingy:91 is a battery-operated prototyping platform for cellular IoT systems, designed to be suited for asset tracking applications and environmental monitoring.
+
+Thingy:91 integrates the following:
+
+* nRF9160 SiP - supporting LTE-M, NB-IoT and Global Positioning System (GPS)
+* nRF52840 SoC - supporting Bluetooth Low Energy and Near Field Communication (NFC)
+
+You can find more information on the product in the `Thingy:91 product description`_ page and in the `Nordic Thingy:91 User Guide`_.
+
+Firmware
+********
+
+The firmware of Thingy:91 has been developed using the nRF Connect SDK.
+It is open source, and can be modified according to specific needs.
+The asset tracker application firmware, which is pre-flashed in the Thingy:91, enables the device to use the environment sensors and provides an option of tracking the device using GPS.
+
+The data, along with information about the device, is transmitted to Nordic Semiconductor's cloud solution, `nRF Cloud`_, where it can be visualized.
+See :ref:`asset_tracker` for more information on the asset tracker application.
+
+Operating modes
+===============
+
+Thingy:91 contains RGB indicator LEDs, which indicate the operating state of the device as described in :ref:`operating states of Thingy:91<thingy91_operating_states>`.
+
+GPS
+===
+
+Thingy:91 has GPS, which if activated, allows the device to be located globally using GPS signals.
+See :ref:`Button SW3 on Thingy:91<asset_tracker_user_interface>` for information on the activation of GPS button on Thingy:91.
+
+LTE Band Lock
+=============
+
+The modem within Thingy:91 can be configured to use specific LTE bands by using the band lock AT command.
+See `nRF9160 Band lock`_ and `band lock section in the AT Commands reference document`_ for additional information.
+The pre-flashed firmware configures the modem to use the bands currently certified on the Thingy:91 hardware.
+The bands to be enabled can be configured when building the firmware.
+
+LTE-M / NB-IoT switching
+========================
+
+Thingy:91 has a multimode modem, which enables it to support automatic switching between LTE-M and NB-IoT.
+A built-in parameter in the Thingy:91 firmware determines whether the modem first attempts to connect in LTE-M or NB-IoT mode.
+If the modem fails to connect using this preferred mode within the default timeout period (10 minutes), the modem switches to the other mode.
+
+Modem firmware
+==============
+
+The functionality of the multimode modem of Thingy:91 is based on its firmware which is available as a pre-compiled binary.
+You can download the modem firmware from the `Thingy:91 product website (downloads)`_.
+The downloadable zip file contains the full firmware image and the firmware image files for updating the firmware.
```

Comment:
@joakimtoe Does the Thingy support incremental updates?

Reply:
I would say something like: "The downloadable zip file contains both the full firmware images used with an external debug probe and the firmware images used for updating the firmware through the built in bootloader"

Reply:
Modified.